media-gfx/blender/files/blender-2.72b-gcc6-fixes.patch

```diff
@@ -0,0 +1,126 @@
+diff -purN a/source/blender/imbuf/intern/dds/ColorBlock.cpp b/source/blender/imbuf/intern/dds/ColorBlock.cpp
+--- a/source/blender/imbuf/intern/dds/ColorBlock.cpp	2014-10-20 08:58:23.000000000 +0100
++++ b/source/blender/imbuf/intern/dds/ColorBlock.cpp	2016-09-22 15:50:25.359318967 +0100
+@@ -86,8 +86,8 @@ void ColorBlock::init(const Image *img,
+ 
+ void ColorBlock::init(uint w, uint h, const uint *data, uint x, uint y)
+ {
+-	const uint bw = min(w - x, 4U);
+-	const uint bh = min(h - y, 4U);
++	const uint bw = MIN(w - x, 4U);
```

Comment:
instead of introducing a macro, why don't you just fully qualify `min` (and `max`) using `std::`? Overzealous use of macros is generally shunned in C++.